src/API.js

```diff
@@ -36,6 +36,19 @@ export const getAllArticlesPagnated = (pageNumber) => {
     })
 }
 
+export const patchArticle = (article_id, up) => {
+    if(up) {
```

Comment:
the `up` logic is a little hard to read for me here. You could just pass the number that you want to change the votes by and then invoked the api function with `1` or `-1`